Research/nfl_model.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import numpy as np
+from sklearn.ensemble import BaggingClassifier, RandomForestClassifier
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.svm import SVC
+
+seasons = pd.read_csv("Data/RegularSeasonData/pbp2009.csv")
+for i in range(7):
```

Comment:
Probably makes sense to have the entirety of your loading operation in a function. The function can take the path to the csv files and the number of files to go over so it's more generic.